### ISSUE TITLE
The hyperparameter setting under BJER4 scenario

### ISSUE THREAD

**Issue by CYBruce** [VeritasYin/STGCN_IJCAI-18#43]
Really appreciate your excellent work on the traffic prediction topic and open source efforts. I am trying to reproduce the performance of your paper on a dataset really similar to BJER4. I changed the essential hyperparameters and the code ran successfully. However, I cannot achieve a satisfactory result and the MAE is even worse than simple FNN and XGboost algorithm.
So my issue is that is there any settings that are really important need to be changed if the road network is smaller than Pems dataset?

**Comment by VeritasYin** [comment]
For dataset like BJER4, it has relatively smaller graph size. The key issue here you may consider is the threshold for generating weighted matrix, and how to prevent over-fitting. Also, MAE probability is not a good metric for optimizing the model.
